molecule/services/tests/test_services.py

```diff
@@ -47,7 +48,7 @@ def test_service(host):
     assert s.is_running
 
 
-@pytest.mark.parametrize("port", [9009, 9010, 9095, 9096])
+@pytest.mark.parametrize("port", [9009, 9010, 9096])
```

Comment:
why is that one gone?

Reply:
This port is not being used in the playbook.